Better use `pkg-config --cflags libopenjp2`

Michael Weber, xmw@gentoo.org

--- leptonica-1.71/src/libversions.c
+++ leptonica-1.71/src/libversions.c
@@ -69,7 +69,7 @@
 #endif
 
 #if HAVE_LIBJP2K  /* assuming it's 2.1 */
-#include "openjpeg-2.1/openjpeg.h"
+#include "openjpeg-2.0/openjpeg.h"
 #endif
 
 #define stringJoinInPlace(s1, s2) \
--- leptonica-1.71/src/jp2kio.c
+++ leptonica-1.71/src/jp2kio.c
@@ -108,7 +108,7 @@
 
     /* Leptonica supports both 2.0 and 2.1.  If you have 2.0,
      * change MINOR to 0.  */
-#define  MINOR   1
+#define  MINOR   0
 
 #if MINOR == 0
   static const l_int32  OpjMinor = 0;
